serverlessworkflow/antora.yml

```diff
@@ -20,3 +20,8 @@ asciidoc:
     cloud_events_sdk_url: https://github.com/cloudevents/sdk-java
     open_api_spec_url: https://spec.openapis.org/oas/v3.1.0.html
     quarkus_openapi_gen_url: https://github.com/quarkiverse/quarkus-openapi-generator
+    java_install: https://www.java.com/en/download/help/download_options.html
```

Comment:
```suggestion
    java_install_url: https://www.java.com/en/download/help/download_options.html
```

Reply:
append the _url suffix to other envs.

Reply:
I just saw that! I've already made the name fixes, and add two missing urls.